Modules/Tokens/test/Tokens.API.Tests.Integration/Models/Token.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+
+namespace Tokens.API.Tests.Integration.Models;
+public class Token
```

Comment:
Why are there so many properties optional?

Reply:
The response content received from a POST and GET requests are very different. A POST request only returns an Id and an expiresAt properties, while a get request return id, createdBy, createdByDevice, createdAt, expiresAt and a content porperties. To validate the json schema I had to turn those properties optional or I would require two different json schemas for a POST and GET requests.

Reply:
But... then the existence of these properties is not validated correctly.

Reply:
Why are the properties still optional, even though you have a separate CreateTokenResponse now?

Reply:
This was all due to the warnings from VS

Reply:
Which warnings?

Reply:
[CS8618](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/nullable-warnings?f1url=%3FappId%3Droslyn%26k%3Dk(CS8618)#nonnullable-reference-not-initialized) - Non-nullable variable must contain a non-null value when exiting constructor. Consider declaring it as nullable.

Reply:
You can leave them unassigned. We have this problem in several places all over the codebase. We will have to find a way to fix that in a pretty way some day.